Lib/test/test_sys.py

```diff
@@ -693,7 +693,7 @@ def test_43581(self):
 
     def test_intern(self):
         self.assertRaises(TypeError, sys.intern)
-        s = "never interned before" + str(random.int(10**8, 10**9))
+        s = "never interned before" + str(random.randint(10**8, 10**9))
```

Comment:
If the intention is to generate random 9-digit numbers, use `randrange()` instead of `randint()`.